localstack/testing/snapshots/transformer_utility.py

```diff
@@ -382,6 +385,24 @@ def sqs_api():
             KeyValueBasedTransformer(_resource_name_transformer, "resource"),
         ]
 
+    @staticmethod
+    def kms_api():
+        """
+        :return: array with Transformers, for kms api.
+        """
+        return [
+            TransformerUtility.key_value("KeyId"),
+            TransformerUtility.jsonpath(
+                jsonpath="$..Signature",
+                value_replacement="<signature>",
+                reference_replacement=False,
+            ),
```

Comment:
that could apparently also just be a key_value transformer since it's just checking for the "Signature" key anyway